website/static/files/build_orca_reconcile.sql

```diff
@@ -6,150 +6,141 @@ SET search_path TO orca, public;
 -- Create the objects in a transaction.
```

Comment:
Actually, I'd vote for deleting that main script. As-is, we have 4 sources of truth for this structure. mdx, xed, sql, db_deploy.

Reply:
I guess I consider db_deploy as the source of truth. The `xed` from ADS is just for helping to design and visualize and really is an artifact for the developers to use to easily work with changes to "show" others in a graphical way. The SQL is really for us to use internally to help use test design without updating db_deploy. I actually use the SQL to create the db_depoy SQL since it is mostly copy/paste.

Reply:
- Then we are okay with the xed going out-of-date?
- What about the mdx?
- If the SQL is used internally, then it still needs to be kept accurate and is thus still a source of truth.

My issue is not in identifying which is our primary source, but that we are presently updating up to 4 different files for each database change.

Reply:
Yes, I think the mdx, xed, and possibly the SQL associated with this mdx should go out of date as we get close to a design and note that it may not reflect the final. I look at them as artifacts from the research spike and would not really look for them to be maintained once the research is done. They are just helpful artifacts used during design process.